main.py

```diff
@@ -1,10 +1,50 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as data:
+        players_data = json.load(data)
+
+    for nickname, player_data in players_data.items():
+
+        race_data = player_data["race"]
+        if Race.objects.filter(name=race_data["name"]).exists():
+            race = Race.objects.get(name=race_data["name"])
+        else:
+            race = Race.objects.create(
+                name=race_data["name"],
+                description=(race_data["description"]
+                             if race_data["description"] else None)
+            )
```

Comment:
you can use here [get_or_create](https://docs.djangoproject.com/en/4.2/ref/models/querysets/#get-or-create)

Reply:
Fix in all places

Reply:
Thx a lot! Done